compose/.env

```diff
@@ -7,38 +11,38 @@
 # The defaults set in this section are also used in the MCPClient container
 # when clamscan is initiated there, rather than the clam daemon.
 
-# Files larger than this limit will not be scanned. The unit used is megabyte 
+# Files larger than this limit will not be scanned. The unit used is megabyte
```

Comment:
Please ignore the diff from here downwards, it's just my editor removing unwanted whitespaces.